c2corg_api/models/user.py

```diff
@@ -103,6 +103,10 @@ class User(Base):
     feed_filter_activities = Column(
         ArrayOfEnum(enums.activity_type), nullable=False, server_default='{}')
 
+    # only show updates from followed users in the homepage feed
+    feed_followed_only = Column(
```

Comment:
No, if `feed_followed_only` is enabled, the area/activities filter are ignored. But you only see changes of your followed users, not the full feed.

To see the full feed, you should not set any area/activities filters.